server/index.mjs

```diff
@@ -391,7 +439,7 @@ const loop = () => {
 
       if (player.settings.fight && player.canAttack({ PF, finder, map })) {
         player.attackDelayTicks = 0;
-        player.energyUse("attack");
+        player.energyUse(ENERGY_ATTACK_USE);
```

Comment:
This is already defined in `server/gameObjects/Player.mjs`
Instead of doing all these actions here it probably would be better to implement `player.attack()` instead.

Additionally maybe attack energy use should be based on current weapon?

Reply:
Yes, I think the same that it should be based on weapon. But it won't be done in this PR